packages/react-native-editor/ios/GutenbergDemo.xcodeproj/project.pbxproj

```diff
@@ -764,7 +764,7 @@
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
```

Comment:
I feel like we've seen this happen before but I'm not sure. I did have some issues with my ruby set up while cutting the release however

Reply:
Yep, we encountered this change in the past. As far as I recall, it was caused by running the command `npm run core preios` from different CPU architecture machines (i.e. x86 vs ARM). I think nowadays we mostly use ARM architectures so I'd say it's a good moment to apply this change 👍 .